recipes/itak/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+cp iTAK.py $PREFIX/bin/
+cp itakm.py ${SP_DIR}
+mkdir $PREFIX/share/itak
+cp -r database $PREFIX/share/itak/
```

Comment:
How big is the database?

Reply:
> How big is the database?

the total size of these database files is around 55mb

Reply:
That is too big sorry. We should not ship databases in packages. Can you please use a post-install script instead?

Reply:
sure, I will add a wget command to download the database from github.

Reply:
> That is too big sorry. We should not ship databases in packages. Can you please use a post-install script instead?

I added a sub-function in Python script for downloading database files, so the database has been removed from the package. Could you please recheck it? please let me know if there is any problem with it. Thank you

Reply:
> How big is the database?

Hi [bgruening](https://github.com/bgruening)

Please let me know how can I solve this problem. Thanks